client/Components/Games/NewGame.jsx

```diff
@@ -100,7 +101,21 @@ class NewGame extends React.Component {
             showHand: this.state.showHand,
             spectators: this.state.spectators,
             useGameTimeLimit: this.state.useGameTimeLimit
-        });
+        };
+
+        if(this.props.tournament) {
+            this.props.matches.forEach(match => {
+                this.props.socket.emit('newgame', {
```

Comment:
It also might be worth having this as a new lobby message and just send the whole array at once

Reply:
v1.1 for sho